### PR TITLE
Netint hardware support

### PR DESCRIPTION
**Pull request by cyberj0g**
**What does this pull request do? Explain your changes. (required)**
NETINT hardware support. Requires corresponding `lpms` version and ffmpeg build.

**Specific updates (required)**
- add `-netint` argument
- refactor startup transcoding test

**How did you test each of these updates (required)**
Tests and manually on Netint server.

**Does this pull request close any open issues?**
No.

**Checklist:**

- [X] Read the [contribution guide](./doc/contributing.md)
- [X] `make` runs successfully
- [X] All tests in `./test.sh` pass
- [ ] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: core/transcoder.go [hunk 1]
```diff
@@ -285,13 +324,68 @@ func TestSoftwareTranscoderCapabilities(tmpdir string) (caps []Capability, fatal
 	return caps, fatalError
 }
 
+func TestNetintTranscoder(devices []string) error {
```
Comment (thomshutt) [inline core/transcoder.go:328]: What calls this method?

Comment (cyberj0g) [reply core/transcoder.go:328]: Removed.

File: cmd/livepeer/livepeer.go [hunk 2]
```diff
@@ -310,22 +311,39 @@ func main() {
 	var transcoderCaps []core.Capability
 	if *transcoder {
 		core.WorkDir = *datadir
+		accel := ffmpeg.Software
+		var devicesStr string
 		if *nvidia != "" {
+			accel = ffmpeg.Nvidia
+			devicesStr = *nvidia
+		}
+		if *netint != "" {
```
Comment (thomshutt) [inline cmd/livepeer/livepeer.go]: Should we make it error if someone tries to pass `-netint` and `-nvidia`?

**Comment by thomshutt** [comment]
Looks good! Not sure if the test failure is temporary or something that needs fixing

**Comment by cyberj0g** [comment]
@thomshutt tests are fixed

File: cmd/livepeer/livepeer.go (hunk 2 above)
Comment (cyberj0g) [reply cmd/livepeer/livepeer.go]: Check added.

**Review by thomshutt — APPROVED** [review]

**Pull request merged**